File: Raven.Client.Java/src/main/java/net/ravendb/client/spatial/SpatialCriteria.java
package net.ravendb.client.spatial;

import net.ravendb.abstractions.indexing.SpatialOptions.SpatialRelation;

public class SpatialCriteria {
  private SpatialRelation relation;
  private Object shape;

  public SpatialRelation getRelation() {
    return relation;
  }
  public void setRelation(SpatialRelation relation) {
    this.relation = relation;
  }
  public Object getShape() {
    return shape;
  }
  public void setShape(Object shape) {
    this.shape = shape;
  }

}

File: src/Raven.NewClient/Spatial/SpatialCriteria.cs
using Raven.Abstractions.Indexing;

namespace Raven.NewClient.Client.Spatial
{
    public class SpatialCriteria
    {
        public SpatialRelation Relation { get; set; }
        public object Shape { get; set; }
        public double DistanceErrorPct { get; set; }
    }
}
